src/partners/changenow.ts

```diff
@@ -64,14 +64,19 @@ export const queryChangeNow = async (
   while (true) {
     const url = makeUrl(settings, pluginParams.apiKeys.changenowApiKey)
     let jsonObj: ReturnType<typeof asChangeNowResult>
+    let retry = 3
     try {
       const result = await fetch(url, {
         method: 'GET'
       })
       const seperate = await result.json()
       jsonObj = asChangeNowResult(seperate)
     } catch (e) {
+      const error: any = e
```

Comment:
You can do `} catch (error: any) {` instead of this. Versions of TypeScript that are new enough to treat errors as `unknown` also support this new syntax for specifying the type. If VScode is saying one thing and `yarn precommit` is saying another, maybe we should upgrade the version of tsc used in the repo.

Reply:
yeah. VSCode auto converts `} catch (error: any) {` into `} catch (error) {` but then complains about `e.code`

I upgraded to the latest typescript and still get the error. Will sidebar to find a fix